src/pykiso/logging_initializer.py

```diff
@@ -234,3 +241,96 @@ def initialize_loggers(loggers: Optional[List[str]]) -> None:
     loggers_to_deactivate = set(relevant_loggers) - set(active_loggers)
     for logger_name in loggers_to_deactivate:
         logging.getLogger(logger_name).setLevel(logging.WARNING)
+
+
+def import_object(path: str) -> Any:
```

Comment:
This feels a bit strange. Do we really want it to import any object? Seems like a security leak somehow. We are in the `logging_initializer.py`. Please make sure we only import the exact needed logging object.

Reply:
A check  to see if the object imported is a subclass of `logging.Logger` has been implemented.

Reply:
It is better, I think you just need to update the return value to `Union[None, logging.Logging]`

Reply:
Done